docs/csharp/reference-semantics-with-value-types.md

```diff
@@ -187,8 +187,7 @@ using `Span<T>` for simplicity.
 
 The `ref struct` declaration declares that a struct of this type must be
 on the stack. The language rules ensure the safe use of these types. Other
-types declared as `ref struct` include `ReadOnlySpan<T>`, `Memory<T>`, and
-`ReadOnlyMemory<T>`. 
+types declared as `ref struct` include `ReadOnlySpan<T>`. 
```

Comment:
nit: "other types" implies multiple types, but only one is present. Perhaps, "For example, the `ReadOnlySpan<T>` type is declared as `ref struct`." (Though personally I like the original version better, even though it seems incomplete by mentioning only one type.)

Reply:
another nit: would be nice to link to the API doc